tests/api/routes/test_events.py

```diff
@@ -0,0 +1,107 @@
+from http import HTTPStatus
+
+import pytest
+from django.utils.timezone import now
+from freezegun import freeze_time
+from hamcrest import (
+    assert_that,
+    empty,
+    equal_to,
+    has_entries,
+    has_length,
+    has_properties,
+    is_,
+)
+
+from meetupselector.proposals.models import Event
+from tests.utils.builders import UserBuilder
+
+
+@freeze_time("2022-10-26 23:23:23")
+@pytest.mark.django_db
+def test_is_staff_create_event(client, reverse_url):
+    url = reverse_url("event")
+    name = "eventName"
+    description = "description"
+    meetup_link = "meetup_link"
```

Comment:
If you use URLField you have to put a valid URL here.

Reply:
Ok. I will change that.

Reply:
Done